app/controllers/students/base_controller.rb

```diff
@@ -8,7 +8,7 @@ def must_be_student
   end
 
   def current_team
-    @current_team ||= current_user.teams.first
+    @current_team ||= current_user.teams.last
```

Comment:
One other thing: do we need to check for the current team where the user assumes a `student` role?